hikari/interactions/component_interactions.py

```diff
@@ -348,6 +353,7 @@ async def fetch_parent_message(self) -> messages.Message:
         """
         return await self.fetch_message(self.message.id)
 
+    @deprecation.deprecated("2.0.0.dev110", "message")
```

Comment:
```suggestion
    @deprecation.deprecated("2.0.0.dev110", "get_message")
```

Reply:
I thought this was all about it being a direct attribute

Reply:
As it is on master atm, that argument takes in an alternative that people should use. In docs v2 branch it is replaced by a "reason" argument, which allows more flexibility. In this case there is a direct alternative, so we should recommend that

Reply:
I assumed the agreed upon reason for deprecation was that the message is already an attribute and getting it from the cache is useless.

Reply:
Oh, didnt even hit me. In that case there is no need for `fetch_message` either (?)

Reply:
`fetch_message` is for the webhook messages but `fetch_parent_message` should probably indeed be deprecated since the potential usecase is very slim